core/src/test/java/com/mapzen/android/graphics/MapzenMapTest.java

```diff
@@ -293,10 +293,10 @@
     verify(overlayManager).onFling(1f, 2f, 3f, 4f);
   }
 
-  @Test public void setRotateResponder_shouldInvokeMapController() {
+  @Test public void setRotateResponder_shouldNotInvokeMapController() {
```

Comment:
This is good but lets also add 2 more tests to ensure that the `OverlayManager` and any external `RotateResponder` objects are invoked:

```
@Test public void setRotateResponder_shouldInvokeOverlayManager() {
    TestRotateResponder rotateResponder = new TestRotateResponder();
    map.setRotateResponder(rotateResponder);
    map.internalRotateResponder.onRotate(1, 2, 0.5f);
    verify(overlayManager).onRotate(1, 2, 0.5f);
  }

  @Test public void setRotateResponder_shouldInvokeRotateResponder() {
    TestRotateResponder rotateResponder = new TestRotateResponder();
    map.setRotateResponder(rotateResponder);
    map.internalRotateResponder.onRotate(1, 2, 0.5f);
    assertThat(rotateResponder.rotated).isTrue();
  }
```

And then update `TestRotateResponder` to look like this:
```
public class TestRotateResponder implements TouchInput.RotateResponder {
  boolean rotated = false;

  @Override public boolean onRotate(float x, float y, float rotation) {
    rotated = true;
    return false;
  }
}
```